client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -352,11 +364,16 @@ export class WorkspaceView extends React.Component<any, any> {
       }
     }
```

Comment:
That's just a prettier commit. The original one looks like Ozzie's here (https://github.com/oughtinc/mosaic/commit/6d2323102d7ee3b5ab7b38715d5e0ab518c274e5) in Mar 2018. It would be beyond impressive if he remembered this.